corehq/apps/data_interfaces/tests/test_auto_case_updates.py

```diff
@@ -334,6 +336,48 @@ def test_boundary_date(self):
             rules_by_case_type['test-case-type-2'], datetime(2016, 1, 1))
         self.assertEqual(boundary_date, datetime(2015, 12, 2))
 
+    @run_with_all_backends
+    def test_parent_case_lookup(self):
+        with _with_case(self.domain, 'test-child-case-type', datetime(2016, 1, 1)) as child, \
+                _with_case(self.domain, 'test-parent-case-type', datetime(2016, 1, 1)) as parent:
+
+            # Set the parent case relationship
+            parent.name = 'abc'
+            parent.save()
```

Comment:
@snopoke do you recommend i do this differently (with a form submission?) or is it fine as is